0_RootFS/gcc_common.jl

```diff
@@ -357,11 +357,18 @@ function gcc_script(compiler_target::Platform)
             GLIBC_CONFIGURE_OVERRIDES+=( libc_cv_ssp=no libc_cv_ssp_strong=no )
         fi
 
-        if [[ ${COMPILER_TARGET} == riscv64-* ]]; then
-            # Explicitly disable C++
-            # (Disable for all architectures?)
-            GLIBC_CONFIGURE_OVERRIDES+=( CXX=false )
-        fi
+        # Explicitly disable C++.
+        # If we don't do this, glibc will pick up the host C++
+        # compiler (/usr/bin/g++) to build some C++ files. With this
+        # setting, glibc will build C versions of these files instead.
+        GLIBC_CONFIGURE_OVERRIDES+=( CXX=false )
+
+        # These flags are necessary for GCC 14. GCC 14 defaults to a
```

Comment:
Should this be guarded by an if based on the gcc version?

Reply:
Good idea.